Documentation/OutputFormats.rst

```diff
@@ -0,0 +1,20 @@
+Output Formats
+=============
+
+ROOT
```

Comment:
Definitely need to expand this section, but not necessarily as part of this PR.

Reply:
added a TODO_DOC flag in c35796d